modules/configuring-default-seccomp-profile.adoc

```diff
@@ -1,15 +1,106 @@
++
+[source, terminal]
+----
+$ oc get pods -n workshop 
+---- 
++
+.Example output 
++
+[source, terminal]
+----
+NAME                READY   STATUS      RESTARTS   AGE
+parksmap-1-4xkwf    1/1     Running     0          2m17s
+parksmap-1-deploy   0/1     Completed   0          2m22s
+---- 
++
+.. Inspect the pods: 
++
+[source, terminal]
+----
+$ oc get pod parksmap-1-4xkwf -n workshop -o yaml
+----
++
+.Example output 
++
+[source, terminal]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/network-status: |-
+      [{
+          "name": "openshift-sdn",
+          "interface": "eth0",
+          "ips": [
+              "10.131.0.18"
+          ],
+          "default": true,
+          "dns": {}
+      }]
+    k8s.v1.cni.cncf.io/networks-status: |-
+      [{
+          "name": "openshift-sdn",
+          "interface": "eth0",
+          "ips": [
+              "10.131.0.18"
+          ],
+          "default": true,
+          "dns": {}
+      }]
+    openshift.io/deployment-config.latest-version: "1"
+    openshift.io/deployment-config.name: parksmap
+    openshift.io/deployment.name: parksmap-1
+    openshift.io/generated-by: OpenShiftWebConsole
+    openshift.io/scc: restricted-v2 <1>
+    seccomp.security.alpha.kubernetes.io/pod: runtime/default <2>
+---- 
+<1> The `restricted-v2` SCC is the SCC added by default, if your workload does not have access to a different SCC. 
+<2> Newly created pods in {product-version} will have the seccomp profile configured to `runtime/default` as mandated by the SCC. 
+
+[id="upgraded_cluster_{context}"]
+== Upgraded cluster
+
+In clusters upgraded to {product-version} all authenticated users have access to the `restricted` and `restricted-v2` SCC. 
+
+If a workload is running for example on a {product-title} v4.10 cluster that was admitted by the SCC `restricted` when the cluster is upgraded that workload may get admitted by `restricted-v2` if the workload is able to run with `restricted-v2`. This is because `restricted-v2` is the most restrictive SCC between `restricted` and `restricted-v2`.
+Conversely with a workload that requires `privilegeEscalation: true` this workload will continue to have the `restricted` SCC available for any authenticated user. This is because `restricted-v2` does not allow `privilegeEscalation`.
+
+[id="newly_installed_{context}"]
+== Newly installed cluster
+
+For newly installed {product-title} v4.11 cluster, the `restricted-v2` replaces the `restricted` SCC as an SCC that is available to be used by any authenticated user. A workload with `privilegeEscalation: true`, will not be admitted into the cluster since `restricted-v2` is the only SCC available for authenticated users by default. 
+
+`privilegeEscalation` is a feature that is allowed by `restricted` but not by `restricted-v2`. It is important to note that more features are denied by `restricted-v2` than were allowed by `restricted` SCC.  
```

Comment:
- Avoid starting a sentence with lowercase words. Try: "The feature `privilegeEscalation` is ..."
- Second sentence: Delete "It is important to note that," starting the sentence with "More."